plugin.video.slingtv/resources/lib/login.py

```diff
@@ -0,0 +1,278 @@
+#   Copyright (C) 2018 Lunatixz, eracknaphobia, d21spike
+#
+#
+# This file is part of Sling.TV.
+#
+# Sling.TV is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Sling.TV is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Sling.TV.  If not, see <http://www.gnu.org/licenses/>.
+
+# -*- coding: utf-8 -*-
+import os, sys, time, _strptime, datetime, re, traceback, pytz, calendar, random
+import urlparse, urllib, urllib2, socket, json, requests, base64, inputstreamhelper
+import xbmc, xbmcgui, xbmcplugin, xbmcaddon, xbmcvfs
+from itertools import zip, cycle
```

Comment:
There is no zip function in itertools.
```suggestion
from itertools import cycle
```